src/diagnostics/conservation_diagnostics.jl

```diff
@@ -0,0 +1,143 @@
+# This file is included in Diagnostics.jl
+
+# Diagnostics for computing mass and energy conservation
+# All the diagnostics in this file are scalars, we need to
+# turn them into vectors as the diagnostics need to be mutable
+
+###
+# Total mass of the air (scalar)
+###
+add_diagnostic_variable!(
+    short_name = "massa",
+    long_name = "Total Mass of the Air",
+    units = "kg",
+    compute! = (out, state, cache, time) -> begin
+        if isnothing(out)
+            return [(sum(state.c.ρ))]
```

Comment:
```suggestion
            return [sum(state.c.ρ)]
```
Same with the other ones